src/StacBrowser.vue

```diff
@@ -49,8 +54,9 @@ import { getBest, prepareSupported } from './locale-id';
 Vue.use(AlertPlugin);
 Vue.use(ButtonGroupPlugin);
 Vue.use(ButtonPlugin);
-Vue.use(BadgePlugin);
 Vue.use(CardPlugin);
+Vue.use(BadgePlugin)
+Vue.use(VueHighlightJS);
```

Comment:
Can this be registered so that it doesn't import into the global bundle? Ideally this would only be loaded if needed.

Reply:
Maybe you could say a bit more - I'm new to Vue, so some of the concepts here require me to squint a bit more than others might

Reply:
I think I follow you - just been using python and jvm lingo for long enough that 'global bundle' tripped me up

Reply:
Global Bundle is probably also not the right terminology. I think better is "app bundle" or "initially loaded files". I pretty much want that it is lazily loaded when needed, not everytime the page loads. :-)